plotly/tests/test_core/test_graph_objs/test_update.py

```diff
@@ -28,7 +28,6 @@ def test_update_dict_empty():
     trace2 = Scatter(x=[1, 2, 3], y=[3, 2, 1])
     data = Data([trace1, trace2])
     data.update({})
-    print(data.to_string())
```

Comment:
🚨 - Breaking Change

I didn't know we had a `to_string()` method. I suppose that we're dropping it? It doesn't seem that useful to me, as you should be able to just do `str(data)`

Reply:
One thing that we still need to talk about is what we should do for the `__repr__`, and `__str__` methods on `Figure` and `BasePlotlyType` objects.